Author: Alexander Grund
Ant 1.10.2 removed the -Xdoclint:none option which leads to build failures
due to optional packages not beeing found during Javadoc generation
See https://bz.apache.org/bugzilla/show_bug.cgi?id=63438
diff -aur a/build.xml b/build.xml
--- a/build.xml	2018-02-03 17:52:24.000000000 +0100
+++ b/build.xml	2020-01-10 13:34:56.885004000 +0100
@@ -1456,6 +1456,7 @@
           description="--> creates the API documentation" unless="javadoc.notrequired">
     <mkdir dir="${build.javadocs}"/>
     <javadoc useexternalfile="yes"
+             additionalparam="-Xdoclint:none"
              destdir="${build.javadocs}"
              failonerror="true"
              author="true"
